### PR TITLE
Add coco to check that integrate_odes() is called if ODEs are defined

### LINKED ISSUE #573: Add coco to log an error if equations defined but integrate_odes() never called

### PR DESCRIPTION
**Pull request by clinssen**
Fixes #573.

### CODE REVIEW

File: pynestml/utils/messages.py [hunk 1]
```diff
@@ -1162,3 +1163,8 @@ def get_could_not_determine_cond_based(cls, type_str, name):
     def get_no_files_in_input_path(cls, path: str):
         message = "No files found matching '*.nestml' in provided input path '" + path + "'"
         return MessageCode.NO_FILES_IN_INPUT_PATH, message
+
+    @classmethod
+    def get_equtions_defined_but_integrate_odes_not_called(cls):
```
Comment (pnbabu) [inline pynestml/utils/messages.py:1168]:
```suggestion
    def get_equations_defined_but_integrate_odes_not_called(cls):
```

**Review by pnbabu — COMMENTED** [review]
A minor comment

**Review by pnbabu — APPROVED** [review]
I am fine with the changes. There are conflicts after the merge of #634. You could resolve them and merge this.

**Comment by clinssen** [comment]
Thanks for the review!

**Pull request merged**